service/controller/app/app.go

```diff
@@ -104,11 +103,8 @@ func NewApp(config Config) (*App, error) {
 			K8sClient:    config.K8sClient,
 			Logger:       config.Logger,
 			ResyncPeriod: config.ResyncPeriod,
-			Pause: map[string]string{
-				annotation.AppOperatorPaused: "true",
-			},
-			Resources: resources,
-			Selector:  label.AppVersionSelector(config.UniqueApp),
+			Resources:    resources,
+			Selector:     label.AppVersionSelector(config.UniqueApp),
```

Comment:
I don't think we should remove that. It's enough if we remove adding pause annotation in app-admission-controller.

But I leave the final say to @giantswarm/team-batman-engineers

Reply:
Yes there could be other situations where we want to use the pause annotation. So I agree with @kopiczko about keeping it here and just removing from app-admission-controller.